task/01-strings-tasks.js

```diff
@@ -69,7 +69,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    return value.substring(7, value.length - 1);
 }
```

Comment:
looks fine, please note, for the substring you can also use negative numbers. `substring(7, - 1)` do the same